docs/devel/translations.rst

```diff
@@ -17,10 +17,35 @@ This can be done using :guilabel:`Start new translation` in :ref:`component`.
 
 .. note::
 
-    Project admins can always start translation within Weblate directly.
+    Project admins can always start translations within Weblate directly.
 
 Language files added manually to the VCS are added to the component when Weblate updates
-the repository. About repository update settings, see :ref:`update-vcs`.
+the repository. ref:`update-vcs` has info on updating repository settings.
+
+.. _removing-translation:
+
+Removing existing translations
+------------------------------
+
+Components or the projects they are in can be removed in their entirety
+from :guilabel:`Removal` found in the administration tab shown to administrators
+of each project or component. Languages can be removed by navigating to the
+one to delete and finding the menu under Tools > Removal.
+
+A list of components to remove is shown, and typing in the entire name of
+the respective project, component or language slated for removal will
+confirm its deletion.
```

Comment:
* The phrase looks a bit weird to me
* `the entire name` is called `slug` in the UI. This term should be used here and defined as it might be unintelligible to some users. An example might be appropriate. Also see the [discussion in comments](https://hosted.weblate.org/translate/weblate/application/ru/?checksum=26bf883744fcd5e9&sort_by=-priority,position#comments).

Proposed text:

```
Upon invoking the :guilabel:`Removal` action the list of components to be removed will be
shown. To Confirm the deletion you have to enter the object's `slug`. The `slug` is the project's, language's or component's pathname as shown in the URL. e.g. for a component ``bar`` of a project ``foo`` the slug would be foo/bar.
```

Reply:
included.